src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
The best practice would be if you take out random as a class variable.
Example: 
![image](https://user-images.githubusercontent.com/12864128/207862942-6121bf38-8a18-4a4e-9c5a-bd84aac5d97e.png)